cdhweb/static_src/global/mixins/three-dee-box.scss

```diff
@@ -0,0 +1,69 @@
+/**
+    Fancy 3D-lookin' box.
+    - Default look: white content area, teal border/sides (overridable). 
+    - Padding of the content is the responsibility of the component that uses this mixin.
+    - Ditto for any change to the size of border/size by breakpoint.
+*/
+
+@mixin three-dee-box {
+  // "Side" = the chonky 3d side, made from a clip-path'd div absolute-positioned behind the content.
+  // "Border" = the conventional border containing the content.
+  // Note: currently no requirement for the border and the sides to be different colour from each other.
+  --side-size: var(--three-dee-side-size, 18px);
+  --border-width: var(--three-dee-border-width, 8px);
+  --sides-color: var(--three-dee-box-border-color, var(--color-brand-100));
+  --content-bg: var(--three-dee-box-content-bg, var(--color-white));
```

Comment:
I haven't gotten to it yet, but this is an example of a component that will use different bg/border colors

<img width="785" alt="Screenshot 2024-05-27 at 9 39 33 AM" src="https://github.com/springload/cdh-web/assets/1134713/9f81b223-03a6-4760-acbb-754963baee51">